.idea/modules.xml

```diff
@@ -5,6 +5,7 @@
       <module fileurl="file://$PROJECT_DIR$/Backpack/Backpack.iml" filepath="$PROJECT_DIR$/Backpack/Backpack.iml" />
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
       <module fileurl="file://$PROJECT_DIR$/backpack-android.iml" filepath="$PROJECT_DIR$/backpack-android.iml" />
+      <module fileurl="file://$PROJECT_DIR$/backpack-android.iml" filepath="$PROJECT_DIR$/backpack-android.iml" />
```

Comment:
any idea where this is from? is not different from the one above

Reply:
Yeah, no idea! I'll just remove it.

Reply:
Actually shouldn't `.idea` just be gitignored?

Reply:
I tought so too, but the default android app has just some subfolders in the gitignore, not really sure what the deal it